src/utils/voting.js

```diff
@@ -1,29 +1,55 @@
+import i18next from 'i18next';
 import votingConst from '../constants/voting';
+import Fees from '../constants/fees';
 
-const getVotedList = votes => (Object.keys(votes).filter(key => votes[key].confirmed));
+export const getVotedList = votes => (Object.keys(votes).filter(key => votes[key].confirmed));
 
-const getUnvoteList = votes => (Object.keys(votes).filter(key =>
+export const getUnvoteList = votes => (Object.keys(votes).filter(key =>
   !votes[key].unconfirmed && votes[key].confirmed));
 
-const getVoteList = votes => (Object.keys(votes).filter(key =>
+export const getVoteList = votes => (Object.keys(votes).filter(key =>
   votes[key].unconfirmed && !votes[key].confirmed));
 
-const getTotalVotesCount = votes => ((getVotedList(votes).length - getUnvoteList(votes).length)
-  + getVoteList(votes).length);
+export const getTotalVotesCount = votes => (
+  (getVotedList(votes).length - getUnvoteList(votes).length)
+  + getVoteList(votes).length
+);
 
-const getTotalActions = votes => (
+export const getTotalActions = votes => (
   Math.ceil((
     getVoteList(votes).length + getUnvoteList(votes).length
   ) / votingConst.maxCountOfVotesInOneTurn)
 );
 
-const getPendingVotesList = votes => (Object.keys(votes).filter(key => votes[key].pending));
+export const getPendingVotesList = votes => (Object.keys(votes).filter(key => votes[key].pending));
+
+export const getVotingLists = votes => ({
+  votedList: getVoteList(votes).map(vote => votes[vote].publicKey),
+  unvotedList: getUnvoteList(votes).map(vote => votes[vote].publicKey),
+});
+
+export const getVotingError = (votes, account) => {
+  let error;
+  if (account.balance < Fees.vote) {
+    error = i18next.t('Not enough LSK to pay for the transaction.');
+  } else if (getTotalVotesCount(votes) > votingConst.maxCountOfVotes) {
+    error = i18next.t('Max amount of delegates in one voting exceeded.');
+  }
+  return error;
```

Comment:
I think you dont need the error variable, maybe something like this.

```
if (account.balance < Fees.vote) return i18next.t('Not enough LSK to pay for the transaction.');
if (getTotalVotesCount(votes) > votingConst.maxCountOfVotes) return i18next.t('Max amount of delegates in one voting exceeded.');
```

but is just a suggestion if not like this is ok too

Reply:
Your suggestion results in eslint error: `consistent-return: Expected to return a value at the end of arrow function.`